README.md

```diff
@@ -1,3 +1,6 @@
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```

Comment:
Missing Azure badge.

Reply:
Also, please add copyright notice at the top of the azure-pipelines.yml file.

Reply:
It's missing the Azure badge because the pipeline hasn't been set up yet, as the repo can't be selected from within Azure. I think you said I need someone with admin ability to add it.